crates/grafbase-sdk/src/component/state.rs

```diff
@@ -14,19 +14,32 @@ type InitFn =
 static mut INIT_FN: Option<InitFn> = None;
 static mut EXTENSION: Option<Box<dyn AnyExtension>> = None;
 static mut SUBSCRIPTION: Option<Box<dyn Subscription>> = None;
+static mut AUDIT_LOGS: Option<wit::AuditLogs> = None;
```

Comment:
The idea, what we had in mind with @Finistere and what I'm going to try out is that:

- ok, audit logs are per request
- we have them as an atomic pointer in the host
- whenever we call an extension, we swap the pointer first
- unsafe cowboy things

The alternative idea is to pass them in every extension call, but I'm trying this approach first.

Reply:
I'll consider in the next PRs.